src/Microsoft.VisualStudio.Threading.Analyzers/VSTHRD002UseJtfRunCodeFixWithAwait.cs

```diff
@@ -59,7 +59,7 @@ public class VSTHRD002UseJtfRunCodeFixWithAwait : CodeFixProvider
 
                             return document.Project.Solution;
                         },
-                        VSTHRD002UseJtfRunAnalyzer.Id),
+                        "only action"),
```

Comment:
we usually use title.

Reply:
That seems reasonable. But as it doesn't appear to show up to the user anywhere, I think a static string should work too.

Reply:
yep. it works. I am just saying title since title usually make debugging a bit easier since I don't need to dig but know right away where this fix is for most of time.

Reply:
Interesting. That does sound potentially useful. Where would I be debugging and see this value (presumably amidst other instances with different values)?

Reply:
...and wouldn't the `CodeAction.Title` also be available to me in that context and be just as easy to read?

Reply:
This string just defines an `equivalence class` of code actions that fix same diagnostic IDs in an identical fashion, and hence can be batched with FixAll. We recommend title as it generally provides clarity for the equivalence class, but any unique key will work.

Reply:
not thing wrong with using "only action". for me, I usually just look equivalent key when debugging fix all. might be only because I am used to it. and yes, you can get title from codeaction if you have codeaction in hand.